src/services/item/plugins/publication/publicationState/test/fixtures.ts

```diff
@@ -0,0 +1,55 @@
+import { v4 } from 'uuid';
+
+import {
+  ItemTagType,
+  ItemType,
+  ItemValidationProcess,
+  ItemValidationStatus,
+  buildPathFromIds,
+} from '@graasp/sdk';
+
+import { ItemTag } from '../../../itemTag/ItemTag';
+import { ItemMetadata, ItemValidationGroupStatus } from '../types';
+
+export const ItemMetadataFactory = (
+  item?: Partial<ItemMetadata> & { parentItem?: ItemMetadata },
+  isPublic?: boolean,
+): ItemMetadata => {
+  const path = item?.path ?? v4();
+
+  return {
+    updatedAt: item?.updatedAt ?? new Date(),
+    path: item?.parentItem ? buildPathFromIds[(item.parentItem.path, path)] : path,
+    type: item?.type ?? ItemType.FOLDER,
+    public: isPublic ? ({ type: ItemTagType.Public } as ItemTag) : undefined,
+  };
+};
```

Comment:
It's because I was unable to use the `PackedFolderItemFactory`. When I used it I had issues because the service wanted a "backend" `PackedItem` and I provided an "SDK" `PackedItem`. I spent a lot of time trying to work with these two types and I finally decided to use a specific type.